src/background.js

```diff
@@ -65,10 +66,21 @@ chrome.tabs.onActivated.addListener(async activeInfo => {
   await writeToLocalStorage({ url })
 })
 
-chrome.alarms.create({
-  periodInMinutes: CHECK_INTERVAL_TIMER_MINUTES,
-  when: Date.now() + 1
-})
+export const createChromeAlarm = async checkIntervalTimerMinutes => {
+  await chrome.alarms.clearAll()
+  chrome.alarms.create({
+    periodInMinutes: Number(checkIntervalTimerMinutes),
```

Comment:
The conversion needs to happen, as stringified numbers are not accepted by the property, either here on the function or in the parameter when it is executed.  I thought it was better here. What do you think?

Reply:
why would we get a stringified number here in the first place?

Reply:
We would not if I set the type of the input to number. Didn't remember the text field in material UI also received the normal input parameters. This would also remove the necessity of validation. Thanks!

Reply:
the point is that this function shouldn't care where input is coming from. to be entirely correct it could do validation to check if it's a number and throw otherwise, but let's say it's the responsibility of the caller to provide valid input